# IMU.msg

Header header
uint16 lDirCmd
uint16 rDirCmd
uint16 lDutyCmd
uint16 rDutyCmd
uint16 lSpeedAct
uint16 rSpeedAct
uint16 pingDist
int16 imuXAccel
int16 imuYAccel
int16 imuZAccel
int32 lPos
int32 rPos

